.gitignore

```diff
@@ -17,6 +17,7 @@ p4_artifacts*
 # Editor config
 .vscode
 .dir-locals.el
+bacon.toml
```

Comment:
Should this be removed?

Reply:
I had been using it (and it is a tool, but find to remove it).

Reply:
It's harmless.  I just wasn't sure if you meant to leave it in or not.  If you want to keep it, feel free.